src/components/LoadingAnimation.scss

```diff
@@ -0,0 +1,128 @@
+@import 'src/styles/settings/general';
+
+@keyframes loadingIcon-opacity {
+    0% {
+        -webkit-transform: translate(0 0);
```

Comment:
I don't think we need vendor prefixes in our codebase. Aren't they being added by some Post CSS-like process? (Also for the `@keyframes`)